scss/_base_code.scss

```diff
@@ -16,10 +16,15 @@
     hyphens: none;
     line-height: 1.5;
     margin-bottom: 0;
+    margin-top: 0;
     tab-size: 4;
     white-space: pre-wrap;
     word-spacing: normal;
     word-wrap: break-word;
+
+    & + & {
+      @include default-vertical-spacing;
```

Comment:
We will need to move this mixin to a global setting to have it available in a base and patterns.